packages/backend/src/scheduler.ts

```diff
@@ -0,0 +1,32 @@
+import * as Sentry from '@sentry/node';
+import { lightdashConfig } from './config/lightdashConfig';
+import Logger from './logger';
+import { SchedulerWorker } from './scheduler/SchedulerWorker';
+import { VERSION } from './version';
+
+process
+    .on('unhandledRejection', (reason, p) => {
+        Logger.error('Unhandled Rejection at Promise', reason, p);
+    })
+    .on('uncaughtException', (err) => {
+        Logger.error('Uncaught Exception thrown', err);
+        process.exit(1);
+    });
+
+Sentry.init({
+    release: VERSION,
+    dsn: process.env.SENTRY_DSN,
+    environment:
+        process.env.NODE_ENV === 'development'
+            ? 'development'
+            : lightdashConfig.mode,
+    integrations: [],
+    ignoreErrors: ['WarehouseQueryError', 'FieldReferenceError'],
```

Comment:
are those supposed to happen inside a scheduler?

Reply:
Just copied this from the backend, I don't think those can happen right now, but maybe if we eventually add a function like compile project. I think it is ok to leave it there, so we don't need to remember adding these in the future